auth.go

```diff
@@ -996,7 +1004,54 @@ func (x *Central) GetAuthenticatorIdentities(getIdentitiesFlag GetIdentitiesFlag
 func (x *Central) GetRoleGroupDB() RoleGroupDB {
 	return x.roleGroupDB
 }
+func RemoveGroupFromAllUsers(c *Central, groupIDString string) error {
+	gID, err := strconv.ParseUint(groupIDString, 10, 32)
+	if err != nil {
+		return err
+	}
+	groupID := GroupIDU32(gID)
+
+	users, err := c.GetAuthenticatorIdentities(GetIdentitiesFlagDeleted)
+	if err != nil {
+		return err
+	}
+
+	for _, user := range users {
+		perm, err := c.GetPermit(user.UserId)
+		if err != nil {
+			continue
+		}
 
+		groups, err := DecodePermit(perm.Roles)
+		if err != nil {
+			return err
+		}
+
+		// Remove the group from the list of groups
+		modified := false
+		for i, g := range groups {
+			if g == groupID {
+				groups = append(groups[:i], groups[i+1:]...)
+				modified = true
+			}
+		}
+
+		// Encode the permit
+		if modified {
+			perm.Roles = EncodePermit(groups)
+			fmt.Printf("Removing group %v: setting permit for %v\n", groupID, user.Email)
```

Comment:
I would rather make use of `c.Log.Debugf`. But Infof might also be fine, as I don't see something like this filling up a log file, or standard out.

Reply:
Keeping it as Infof for now. We still look at the auth logfile very regularly to check if groups have been recognized and appropriate action has been taken on the user.